Translation/en/RedirectedResources/assets/abdata/action/list/shopinfo/30/30/translation.txt

```diff
@@ -1,21 +1,18 @@
-//
-// Dumped for Koikatsu Sunshine v1.0.1 (KoikatsuSunshine) by KKS_TextDump v1.4.5.0
-//
```

Comment:
You need to keep these headers when translating since they're used by the dump tool to track dump version.

Reply:
Headers have been re-added back into the files.